libopenage/engine.cpp

```diff
@@ -548,4 +542,12 @@ void Engine::move_phys_camera(float x, float y, float amount) {
 	this->engine_coord_data->camgame_phys += cam_delta.to_phys3();
 }
 
+void Engine::start_game(std::unique_ptr<GameMain> game) {
+	// TODO: maybe implement a proper 1-to-1 connection
+	assert(game);
```

Comment:
ok, but why don't we just leave it in? or do you wanna optimize the branch out? I mean even in the "release" build those errors should be detected and reportable I think.